sql/src/test/groovy/io/crate/integrationtests/TableFunctionITest.groovy

```diff
@@ -88,4 +88,41 @@ class TableFunctionITest extends SQLTransportIntegrationTest {
         execute("select col1 from unnest([1, 2]) where col1 = 2")
         assert printedTable(response.rows()) == "2\n"
     }
+
+
+
+
+    @Test
+    public void testSelectFromGenerateSeries() throws Exception {
```

Comment:
I don't think we need any extra integration tests. The unit tests should be enough.